### PR TITLE
Corrects a few cargo crates/exports (and nerfs some)

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Requests

Cargo crates in the medical file rejoice you now have a lockable crate to go with your access locked crates.
Russian gear costs more, the armory one now has 5 items rather then 10

## Why It's Good For The Game

Russian gear creep has gotton out of hand, and needs a major relook into balance.

## Changelog
:cl:
add: Medical locked crates
balance: Russian gear crates have less gear in some cases but all will cost more
fix: Medical locked crates that were not locked
spellcheck: Crates that were out of date are corrected
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/emergency.dm [hunk 1]
```diff
@@ -130,7 +130,7 @@
 
 /datum/supply_pack/emergency/bomb
 	name = "Explosive Emergency Crate"
-	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station des... I mean needs! (Time not included.)"
+	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station kills-... I mean needs! Time not included, but what is a full bomb suit and hood, gas mask with the tools you need to defuse a none-nuclear bomb."
```
Comment (Detective-Google) [inline code/modules/cargo/packs/emergency.dm]: `desc = "Science gone bonkers? Beeping behind the airlock? Buy now and become the hero the station ki-... I mean needs! Time not included, but a full bomb suit and hood, as well as a mask and defusal kit are! Non-Nuclear ordinance only.` 

this might be better? the current line you propose has a few spelling errors so at the very least fix those.

File: code/modules/cargo/packs/armory.dm [hunk 2]
```diff
@@ -193,11 +193,11 @@
 					/obj/item/clothing/head/helmet/rus_ushanka,
 					/obj/item/clothing/suit/armor/vest/russian_coat,
 					/obj/item/gun/ballistic/shotgun/boltaction,
-					/obj/item/gun/ballistic/shotgun/boltaction)
+					/obj/item/staff/broom) //Its... A real thing that was done in the Russian army
```
Comment (Ghommie) [inline code/modules/cargo/packs/armory.dm]: Eeeh, broom brigades were not restricted to that country, you may want to modify the comment, or even remove this line cause people may not understand the reference.

File: code/modules/cargo/packs/emergency.dm [hunk 3]
```diff
@@ -130,7 +130,7 @@
 
 /datum/supply_pack/emergency/bomb
 	name = "Explosive Emergency Crate"
-	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station des... I mean needs! (Time not included.)"
+	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and become the hero the station ki-... I mean needs! Time not included, but a full bomb suit and hood, as well as a mask and defusal kit are! Non-Nuclear ordinance only."
```
Comment (Ghommie) [inline code/modules/cargo/packs/emergency.dm]:
```suggestion
	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and be the hero the station des... I mean needs! (Time not included. Not meant for nuclear ordnances.)"
```
Ruining a funny description are we?

File: code/modules/cargo/packs/materials.dm [hunk 4]
```diff
@@ -48,6 +48,22 @@
...
+	crate_name = "plastitanium sheets crate"
+
+/datum/supply_pack/materials/plastitanium_glass50
+	name = "50 Plastitanium Glass Sheets"
+	desc = "Look out from your evil empire with fifty plastitanium glass sheets."
+	cost = 9500
+	contraband = TRUE
+	contains = list(/obj/item/stack/sheet/plastitaniumglass/fifty)
+	crate_name = "plastitanium glass sheets crate"
+
```
Comment (Ghommie) [inline code/modules/cargo/packs/materials.dm]: No, some materials shouldn't be obtainable through cargo as of now.

File: code/modules/cargo/packs/organic.dm [hunk 5]
```diff
@@ -213,7 +214,7 @@
 
 /datum/supply_pack/organic/fakemeat
 	name = "Meat Crate 'Synthetic'"
-	desc = "Run outta meat already? Keep the lizards content with this freezer filled with cruelty-free chemically compounded meat! Contains 12 slabs of meat product, and 4 slabs of *carp*."
+	desc = "Run outta meat already? Keep the lizards content with this freezer filled with cruelty-free chemically compounded meat! Contains 9 slabs of meat product, 3 crab meat and 4 slabs of *carp*."
```
Comment (Ghommie) [inline code/modules/cargo/packs/organic.dm]: Eugh, imitation crab sticks are not really the same as crab meat.

File: code/modules/cargo/packs/organic.dm [hunk 6]
```diff
@@ -224,9 +225,9 @@
 					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
 					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
 					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
-					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
-					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
-					/obj/item/reagent_containers/food/snacks/meat/slab/meatproduct,
+					/obj/item/reagent_containers/food/snacks/meat/rawcrab,
+					/obj/item/reagent_containers/food/snacks/meat/rawcrab,
+					/obj/item/reagent_containers/food/snacks/meat/rawcrab,
```
Comment (Ghommie) [inline code/modules/cargo/packs/organic.dm]: So, you should undo this change until actual crab sticks are coded in.

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/cargo/packs/materials.dm (hunk 4 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs/materials.dm]: Its... Round start...

File: code/modules/cargo/packs/emergency.dm (hunk 3 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs/emergency.dm]: The whole point of the change was to tell people what is in the box.

File: code/modules/cargo/packs/armory.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/modules/cargo/packs/armory.dm]: In the first Grate War, their was not a lot of guns and ammo.
Russians were forced to be in pares of two, one person would have a clip of ammo and a broom to make it look like a gun at  a dissidence. Well the other one had the gun.

File: code/modules/cargo/packs/materials.dm [hunk 7]
```diff
@@ -47,7 +47,23 @@
...
+	crate_name = "plastitanium sheets crate"
+
+/datum/supply_pack/materials/plastitanium_glass50
+	name = "50 Plastitanium Glass Sheets"
+	desc = "Look out from your evil empire with fifty plastitanium glass sheets."
+	cost = 9500
+	contraband = TRUE
+	contains = list(/obj/item/stack/sheet/plastitaniumglass/fifty)
+	crate_name = "plastitanium glass sheets crate"
+*/
```
Comment (Ghommie) [inline code/modules/cargo/packs/materials.dm]: Just remove the now commented code.

File: code/modules/cargo/packs/emergency.dm (hunk 3 above)
Comment (Ghommie) [inline code/modules/cargo/packs/emergency.dm]:
```suggestion
	desc = "Science gone bonkers? Beeping behind the airlock? Buy now and become the hero the station des... I mean needs! Time not included, but a full bomb suit and hood, as well as a mask and defusal kit are! Non-Nuclear ordnances only."
```

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]
